stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -87,10 +87,196 @@ public enum _ExecutionMode : Equatable {
   }
 }
 
+/// TraceContext contains the state needed to build a trace graph function
+/// (TF_Function). As eager ops are executed in tracing mode, their
+/// corresponding nodes are added to the trace graph (via
+/// `addEagerOpToGraph()`). When the trace is finalized (via `finalize()`), the
+/// trace graph function can then be executed (via `execute()`) by the eager
+/// runtime.
+class TraceContext {
```

Comment:
The entire `TraceContext` type can also be `private` because it's not in use elsewhere.

Reply:
Done